Aardvark/Private Categories/UIApplication+ARKAdditions.swift

```diff
@@ -22,16 +22,20 @@ import Foundation
 
 
 extension UIApplication {
-    @nonobjc private static var observingKeyWindowNotifications = false
-    @nonobjc private static let bugReporterToGestureRecognizerMap: NSMapTable<ARKBugReporter, UIGestureRecognizer> = NSMapTable.strongToStrongObjects()
-    
-    @nonobjc func addTwoFingerPressAndHoldGestureRecognizerTrigger(with bugReporter: ARKBugReporter) {
+    @nonobjc
+    private static var observingKeyWindowNotifications = false
+    @nonobjc
```

Comment:
nit: For multi-line declarations, feels like there should be a blank line in between.
```suggestion

    @nonobjc
```

Reply:
Fair! Also: the GitHub suggested changes UI just blew my mind. 👏

Reply:
Yeah, first time I've seen that.  Seems to work pretty well.  👌